openedx/openedx-platform
# feat: add drag-and-drop v2 as bug green button

## Description

Adds a big green button for the drag-and-drop-V2 xblock, as it is no longer hidden in the "problem" section.

## Supporting information

Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
Be sure to check they are publicly readable, or if not, repeat the information here.

## Testing instructions

Checkout
run paver update_assets in the cms shell
open demo course
see the button, use it to create a drag-and-drop-v2 xblock.

## Deadline

"None" if there's no rush, or provide a specific date or event (and reason) if there is one.

## Other information

Include anything else that will help reviewers and consumers understand the change.
- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
- If your [database migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) can't be rolled back easily.
![Screenshot 2023-03-22 at 4 21 45 PM](https://user-images.githubusercontent.com/49422820/227030032-fdce99a7-b16f-4894-86b3-00e444661327.png)
